package.json

```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "clean": "rimraf node_modules"
   },
   "private": true,
   "dependencies": {
```

Comment:
You still have references to Angular 11 here. I think these need to be at least 12 before the build will use Ivy successfully.

Reply:
Yes, you are right. the angular CLI depends upon `ng-packager`, which is now bumped to angular v12 compliant version too.